PlayCover/en.lproj/Localizable.strings

```diff
@@ -113,6 +113,7 @@
 "error.appEncrypted" = "This app is encrypted! Please use a decrypted IPA. iMazing IPAs are not supported!";
 "error.appCorrupted" = "Something went wrong with this IPA. Please try to use another IPA file.";
 "error.appProhibited" = "Using this app through PlayCover will likely result in a ban!";
+"error.appProhibitedTiktok" = "TikTok behaves maliciously on Macs & using it can leave your device in an unbootable state, therefore is blocked from PlayCover. (Also.. Uninstalling this devil's app...)";
```

Comment:
```suggestion
"error.appProhibitedTiktok" = "TikTok behaves maliciously on Macs, and using it can leave your device in an unbootable state, therefore it is blocked from PlayCover.";
```
Although, a more general message would be better.

Reply:
The references should be general, and not specifically referenced as TikTok.

Reply:
we should also inform the user about the uninstallation

Reply:
A rephrase is needed then. In order to make this feature future proof, it must be a general message that could work for more apps.

Reply:
is in it better to just rewrite PROHIBITED_APPS to do the uninstall and message? would solve every issue. instead of creating one specific for tiktok

Reply:
Uninstall should be reserved for malicious apps only. Hence @TheMoonThatRises's suggestion would be better.